src/main/java/school/lemon/changerequest/java/io/hw2/FileCopyUtilImpl_BytePerByte.java

```diff
@@ -0,0 +1,27 @@
+package school.lemon.changerequest.java.io.hw2;
+
+import java.io.*;
+import java.nio.file.FileAlreadyExistsException;
+
+/**
+ * Created by lera on 13.02.17.
+ */
+public class FileCopyUtilImpl_BytePerByte implements FileCopyUtil {
```

Comment:
Please follow Java naming conventions for class/variable/method names.
In your case, this class could have following name: `BytePerByteFileCopyUtil`.

There is no need to append `Impl` each time.

Also, please rename all your implementations to follow java conventions.